### PR TITLE
Modify dockerfile for cAdvisor to build from fork

### PR DESCRIPTION
**Pull request by katarzyna-z**
Signed-off-by: Katarzyna Kujawa <katarzyna.kujawa@intel.com>

### CODE REVIEW

**Comment by katarzyna-z** [comment]
@wacuuu @ppalucki It is ready to review.

File: examples/kubernetes/monitoring/cadvisor/Dockerfile.cadvisor [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
-  yum install -y ipmctl ipmctl-devel
-
+RUN wget https://golang.org/dl/go1.13.14.linux-amd64.tar.gz && \
+  tar -C /usr/local -xzf go1.13.14.linux-amd64.tar.gz && \
+  rm -f go1.13.14.linux-amd64.tar.gz
 
-ADD . /go/src/github.com/google/cadvisor
-WORKDIR /go/src/github.com/google/cadvisor
+ENV PATH=$PATH::/usr/local/go/bin
+RUN git clone --single-branch --branch jwalecki/merged-features https://github.com/wacuuu/cadvisor.git
```
[the remark this answers is not in the document]
Comment (katarzyna-z) [reply examples/kubernetes/monitoring/cadvisor/Dockerfile.cadvisor]: Added `--depth=1`

**Review by ppalucki — APPROVED** [review]
Are you willing to add CI/CD tests now or we will do this in follow up PR?

**Comment by katarzyna-z** [comment]
I prefer to add  tests for cAdvisor in different PR.

**Comment by katarzyna-z** [comment]
@ppalucki @wacuuu docker image for cAdvisor from fork is now available in WCA docker registry

**Review by wacuuu — APPROVED** [review]

**Pull request merged**